src/main/telemetry/telemetry.c

```diff
@@ -48,6 +48,11 @@ static bool telemetryPortIsShared;
 
 static telemetryConfig_t *telemetryConfig;
 
+
+uint8_t useHottAlarmSoundPeriod (void)
+{
+    return telemetryConfig->HottAlarmSoundPeriod;
+}
```

Comment:
this method is not needed, just access the variable directory. 

nothing is being `used` here, this method is just a 'getter' method which we don't need.